api/handlerFuncs.go

```diff
@@ -0,0 +1,101 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type Link struct {
+	Link string `json:"link"`
```

Comment:
Is this enough? Just a link. No full or short form, no reference to its owner/account, no nothing?

Reply:
This method would shorten link in some way, which user don't know. For customizing links we have special method

Reply:
I mean, are you going to save the resulting short link somewhere, in some kind of storage? For that you need some struct field, I guess.